clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -128,6 +145,17 @@
   )
 )
 
+(define-read-only (is-registered-swap-token (swap-token principal))
+  (is-some (map-get? registered-swap-tokens { swap-token: swap-token }))
+)
+
+(define-private (register-swap-token (swap-token principal))
+  (begin
+    (asserts! (is-eq (is-registered-swap-token swap-token) false) (err ERR-WRONG-SWAP-TOKEN))
```

Comment:
```suggestion
    (asserts! (not (is-registered-swap-token swap-token)) (err ERR-WRONG-SWAP-TOKEN))
```
`(not x)` is cheaper to execute than `(is-eq x false)` and I also thing it is more readable.